### DISCUSSION TITLE
No icon

### DISCUSSION THREAD

**Discussion by aluopy** [mmistakes/minimal-mistakes#3659] [Q&A] [answered]
![企业微信截图_16532753861287](https://user-images.githubusercontent.com/69625113/169736002-07d961c6-331b-4011-bc74-1ff6751f5cb0.png)
![企业微信截图_1653274994383](https://user-images.githubusercontent.com/69625113/169736010-0305c1bf-634b-4fc6-8a6e-d3b5dd5ad8b0.png)
How do I need to set it, thank you.

**Comment by mmistakes** [comment, marked as answer]
You have set them correctly and the theme is working as intended. I looked at your site and the icons show for me.

<img width="724" alt="Screen Shot 2022-05-27 at 2 12 15 PM" src="https://user-images.githubusercontent.com/1376749/170767620-6434f5ad-7850-4144-9a01-87b554c227e0.png">

This is a common issue when using some sort of browser extension/plugin that blocks ads, social buttons, and other content.
